snap/snapcraft.yaml

```diff
@@ -877,7 +877,7 @@ parts:
       cd $SNAPCRAFT_PART_INSTALL
       # TODO: SIMPLIFY THIS!!!
```

Comment:
Please remove this comment. It's unclear what is supposed to be simplified, of if it has already been so.

Reply:
Thanks, removed.